docs/user/tutorials/install_epinio_on_eks.md

```diff
@@ -0,0 +1,94 @@
+#  AWS EKS configuration
+
+## Create an EKS cluster
+
+If you don't have an existing cluster, follow the [quickstart](https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html) to create an EKS cluster.
+
+## EKS Prerequisites
+
+* We follow the documented recommendation of having at least 2 nodes for EKS
+* Epinio v0.0.21 has been tested with AWS EKS incl. kubernetes version v1.20.4
+* To do more extensive testing we recommend a 2 node cluster with "t3.xlarge" instances
+* To just try out Epinio, e.g. 2 "t3a.large" are sufficient
+
+#### Install Dependencies
+
+Follow these [steps](./install_dependencies.md) to install dependencies.
+
+#### Install Epinio CLI
+
+* Download the binary
+
+Find the latest version from [Releases](https://github.com/epinio/epinio/releases) and run e.g.
+
+```bash
+curl -o epinio -L https://github.com/epinio/epinio/releases/download/v0.0.20/epinio-linux-amd64
+```
+
+* Make the binary executable
+
+```bash
+chmod +x epinio
+```
+
+* Move the binary to your PATH
+
+```bash
+sudo mv ./epinio /usr/local/bin/epinio
+```
+
+#### Install
+
+Installing Epinio in an EKS cluster will be done in two steps.
+
+#### Install Ingress In Cluster (for a custom DOMAIN)
+
+Install ingress first and wait for the `loadbalancer-ip` to be provisioned for the `traefik` ingress. Then, you can map the `loadbalancer-ip` to your `Domain Name` e.g. `example.net` and wait for it to be mapped.
+
+```bash
+epinio install-ingress
+```
+
+The output of the command will print the `loadbalancer-ip`, but with EKS it will print a loadbalanced FQDN, possibly resolving to multiple IPs. Therefore we recommend to create a wildcard domain using CNAME records.
+
+#### Example wildcard DOMAIN with AWS "route53" service
```

Comment:
I would indent this and its subsections, to make clear that they are part of the first step.

Reply:
True, but the two step install is mentioned in any docs - so that's copied from e.g. GKE, part of AKS, aso. - I'd prefer to have it in another docs fix PR, to keep them similar/close

Reply:
Question of style maybe. With the section at the same level it sort of looks as if there are three steps. Ok.